D1673/P1673.bs

```diff
@@ -7479,8 +7758,8 @@ impossible to express the update A = A - x x^H otherwise.
 For `i,j` in the domain of `A`
 and in the triangle of `A` specified by `t`,
 the mathematical expression for the algorithm is
-`A[i,j] += x[i] * conj(x[j])` for overloads without an `alpha` parameter, and
-`A[i,j] += alpha * x[i] * conj(x[j])` for overloads without an `alpha` parameter.
+`A[i,j] += x[i] * `_`conj-if-needed`_`(x[j])` for overloads without an `alpha` parameter, and
+`A[i,j] += alpha * x[i] * `_`conj-if-needed`_`(x[j])` for overloads without an `alpha` parameter.
```

Comment:
Incidentally, I notice what looks like a copy-paste error here. Should this read 
```suggestion
`A[i,j] += alpha * x[i] * `_`conj-if-needed`_`(x[j])` for overloads with an `alpha` parameter.
```

Reply:
Good catch!  I'll fix this in #242.  Thanks!